gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/upgrades/DomainPasswordPoliciesUpgrader.java

```diff
@@ -43,13 +45,18 @@ public class DomainPasswordPoliciesUpgrader extends SystemTaskUpgrader {
     private static final String TASK_ID = "domain_password_settings_migration";
     private static final String UPGRADE_NOT_SUCCESSFUL_ERROR_MESSAGE =
             "Domain Password Policy can't be upgraded, other instance may process them or an upgrader has failed previously";
+    @SuppressWarnings("squid:S2068") // not a hardcoded password
     public static final String PASSWORD_POLICY_NAME_DEFAULT = "default";
```

Comment:
It would be better to rename is to NAME_DEFAULT. SuppressWarnings will not be needed